.github/workflows/ci.yml

```
@@ -26,3 +26,10 @@ jobs:
 
       - name: Test
         run: cargo test
+
+      - name: Doc Test
+        # Doc-Tests cannot be run on "cdylib" crates.
```

```suggestion
        # MinerSebas: Doc-Tests cannot be run on "cdylib" crates.
```
Okay if we add your username to the comment? Otherwise, LGTM.